### PR TITLE
[16.0][MIG] hotel

### PR DESCRIPTION
**Pull request by sachin-serpentcs**

### CODE REVIEW

File: hotel/__manifest__.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
                Services Pvt. Ltd., OpenERP SA",
     "category": "Hotel Management",
     "website": "https://github.com/OCA/vertical-hotel",
-    "depends": ["sale_stock", "account"],
+    "depends": ["sale_stock", "account", "uom"],
```
Comment (murtuzasaleh) [inline hotel/__manifest__.py]: @pavan-serpentcs Any good reason to add "uom" in dependency?

The stock modules already have dependency products, and the products have the uom module as a dependency.

File: hotel/models/account_move.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+
+    _inherit = "account.move"
+
+    @api.model
```
Comment (Trivedi-Vacha-SerpentCS) [inline hotel/models/account_move.py]:
```suggestion
        @api.model_create_multi
```

**Review by Trivedi-Vacha-SerpentCS — APPROVED** [review]

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

**Review by murtuzasaleh — APPROVED** [review]
LGTM

**Pull request merged**